.github/workflows/check-deps.yml

```diff
@@ -16,7 +16,7 @@ jobs:
         with:
```

Comment:
possibly i think we also want to add permissions for this workflow

Reply:
atm i think this one just needs `issues: write`

Reply:
Done.